### DISCUSSION TITLE
填写了要抢的票的网址，但获取配置未响应

### DISCUSSION THREAD

**Discussion by ximengzhilv** [mikumifa/biliTickerBuy#436] [提问专区]
好奇怪吖，我是按照提示的格式填入的吖
![9f79a28b69f4d35124aa37c7b5527c83](https://github.com/user-attachments/assets/2df42de7-5235-4b99-b54d-3b0980b35997)

**Comment by ximengzhilv** [comment]
现在解决了，大概是网络的缘故，但是过验证码的Amorter该如何添加进配置呢（文件已下好

**Comment by mikumifa** [comment]
选那个就行，程序已经帮你打包好了

**Reply by ximengzhilv** [reply]
这样吗，好的谢谢
